SnmpSharpNet.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio Version 17
```

Comment:
Version 16 (VS 2019) should be the minimal (to support .NET Standard 2.1). No need to jump to 17.